PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/database/RegionComboBox.java

```diff
@@ -65,10 +65,12 @@ protected String getItemText(Object item) {
     }
 
     @Override
-    public void setItem(Region item) {
-        super.setItem(item);
-        if (!Objects.equals(item, getItem())) {
+    public void setSelectedItem(Object value) {
```

Comment:
cannot understand the reason of the changes in this method

Reply:
if value is changed, update the "changed" property.

Reply:
avoid to validate when the region is not changed?

Reply:
yes